dpe/src/commands.rs

```diff
@@ -477,11 +535,22 @@ mod tests {
         }
     }
 
+    impl From<CertifyKeyCmd> for Vec<u8> {
+        fn from(value: CertifyKeyCmd) -> Self {
+            let mut raw = vec![];
+            raw.extend(value.handle);
+            raw.extend_from_slice(&value.flags.to_le_bytes());
+            raw.extend(value.label);
+            raw
```

Comment:
Similarly, those types of function could be `bytemuck::bytes_of(&value).to_vec()`.

Reply:
Replaced with zerocopy